### ISSUE TITLE
Nitpick/Question regarding digital signatures part

### ISSUE THREAD

**Issue by Nicoretti** [exasol/exasol-java-tutorial#23]
**The following sentence:**

https://github.com/exasol/exasol-java-tutorial/blob/c933f26e7867051ad0fac7f38084f973a84eed44/tls-tutorial/doc/tls_introduction.md?plain=1#L133

is a bit overloaded from my point of view. Even though technically I understand that the signature is smaller, I'm not sue if that information is relevant for most readers. I even think it can distract from the more relevant properties (more relevant in the context of the target/reader audience) like that it is tamper-proof. 

**TL;DR:**

* Why is it relevant to the reader that the signature is smaller and how it looks like?
  (Isn't the relevant information just that it is tamper-proof and maybe that it is based on the content?)
